reports/issue-count-cycle-time.ts

```diff
@@ -0,0 +1,83 @@
+import {ProjectData, IssueCard} from '../interfaces';
+import * as rptLib from '../project-reports-lib';
+const tablemark = require('tablemark')
+import * as os from 'os';
+
+export type CycleTimeData = { [key: string]: CycleTimeStageData }
+interface CycleTimeStageData {
+  title: string,
+  cycletime: number,
+  limit: number,
+  count: number,
+  flag: boolean
+}
+
+interface CycleTimeRow {
+    worktype: string,
+    labels: string,
+    count: number,
+    "Cycle Time in days": string,
```

Comment:
I think if you call it cycleTimeInDays, then tablemark will render as "Cycle time in days"